src/util/ConfigManager/ConfigOption.h

```diff
@@ -210,7 +211,7 @@ class ConfigOption {
   @param defaultValue The optional default value. An empty `std::optional` is
   for no default value and a non empty for a default value.
   */
-  template <ad_utility::SameAsAnyTypeIn<ConfigOption::AvailableTypes> T>
+  template <ad_utility::SameAsAnyTypeIn<AvailableTypes> T>
```

Comment:
If you define the concept from line 267 earlier you can also use it here

Reply:
Nah, that is straight up not possible, even with forwarding declaration. (I already tried.)